packages/connect-explorer/src/components/ApiPlayground.tsx

```diff
@@ -39,18 +39,6 @@ const ApiPlaygroundWrapper = styled.div`
     } */
 `;
 
-// eslint-disable-next-line local-rules/no-override-ds-component
-const CollapsibleBoxStyled = styled(CollapsibleBox)`
-    margin: 0;
-    border: 0;
-
-    /* CollapsibleBox content, fix for nested sticky elements */
```

Comment:
We need this to have working scrolling inside the box, can you suggest an alternate solution?

See here for example: 
https://dev.suite.sldev.cz/connect/chore/update-staking-modal/methods/bitcoin/signTransaction/?submethod=multisig
vs desired behavior
https://dev.suite.sldev.cz/connect/develop/methods/bitcoin/signTransaction/?submethod=multisig

Reply:
I will try to think of something:)